Server/Models/chat.js

```diff
@@ -0,0 +1,25 @@
+const { Model, DataTypes } = require('sequelize');
+const sequelize = require('../config/database');
+
+class Chat extends Model {}
+
+Chat.init({
+  id: {
+    type: DataTypes.INTEGER,
+    primaryKey: true,
+    autoIncrement: true,
+    field: 'id',
+  },
+  chatId: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+    field: 'chat_id',
+  },
```

Comment:
why there is an ChatId ?? 
based on your code it is doing nothing 
better delete it